lesson0/homework.js.js

```diff
@@ -0,0 +1,77 @@
+// for(let i = 100; i > 0; i -=2) {
```

Comment:
Rename `lesson0/homework.js.js` to `lesson0/homework.js` to meet the conventions